collectors/freeipmi.plugin/multi_metadata.yaml

```diff
@@ -1,94 +1,26 @@
 name: freeipmi.plugin
 modules:
-  - meta:
-      plugin_name: freeipmi.plugin
-      module_name: sel
-      monitored_instance:
-        name: freeipmi sel
-        link: ''
-        categories: []
-        icon_filename: ''
-      related_resources:
-        integrations:
-          list: []
-      info_provided_to_referring_integrations:
-        description: ''
-      keywords: []
-      most_popular: false
-    overview:
-      data_collection:
-        metrics_description: ''
-        method_description: ''
-      supported_platforms:
-        include: []
-        exclude: []
-      multi-instance: true
-      additional_permissions:
-        description: ''
-      default_behavior:
-        auto_detection:
-          description: ''
-        limits:
-          description: ''
-        performance_impact:
-          description: ''
-    setup:
-      prerequisites:
-        list: []
-      configuration:
-        file:
-          name: ''
-          description: ''
-        options:
-          description: ''
-          folding:
-            title: ''
-            enabled: true
-          list: []
-        examples:
-          folding:
-            enabled: true
-            title: ''
-          list: []
-    troubleshooting:
-      problems:
-        list: []
-    alerts: []
-    metrics:
-      folding:
-        title: Metrics
-        enabled: false
-      description: ""
-      availability: []
-      scopes:
-        - name: global
-          description: ""
-          labels: []
-          metrics:
-            - name: ipmi.sel
-              description: IPMI Events
-              unit: "events"
-              chart_type: area
-              dimensions:
-                - name: events
   - meta:
       plugin_name: freeipmi.plugin
       module_name: sensors
       monitored_instance:
-        name: freeipmi sensors
-        link: ''
-        categories: []
-        icon_filename: ''
+        name: FreeIPMI
```

Comment:
Should this be FreeIPMI? Or just IPMI? 

I mean users care about IPMI don't they?

Reply:
That's a good question, plugin is called `freeipmi`, so I think this is one for @ilyam8 mostly..

Reply:
cc: @ktsaou @sashwathn

Reply:
Name should be IPMI, but we should have a ‘virtual’ integration called ‘FreeIPMI’ as well.

Reply:
right, will spin up an anchor for this, commit incoming.